apps/cms-v2/src/modules/providers/datocms/datocms-client.ts

```diff
@@ -171,11 +190,16 @@ export class DatoCMSClient {
         );
 
         if (isUniqueIdError) {
+          this.logger.warn("Found unique id error, will update the product", {
+            error: isUniqueIdError,
+            variantId: variant.product.id,
+          });
```

Comment:
question: WDYT, I'm a little worried about amount of such cases and then amount of such logs

Reply:
Why this is a warning? It is normal operation when upsert (update or insert)